cypress/support/elements/activity-page.js

```diff
@@ -140,6 +143,12 @@ class ActivityPage {
   getHintText() {
     return this.getInteractive().find('.hint.question-txt');
   }
+  hasHintIcon(interactive) {
+    return cy.wrap(interactive).find('[data-cy=open-hint]', { timeout: 0 });
```

Comment:
Thank you for pointing that out! I added { timeout: 0 } based on a suggestion from ChatGPT to help with some errors I was encountering. This setting prevents Cypress from retrying the find() command if the hint icon isn’t immediately available. It might not be the ideal solution and would welcome any feedback—this was a bit of a ‘quick fix’ on my part.